README.md

```diff
@@ -16,15 +24,26 @@ Both unit and integration tests go in [src/test/java/uk/gov/mca/beacons/service]
 - `./gradlew integrationTest` runs integration tests
 - `./gradlew check` runs both unit and integration tests
 
+## Building 
+
+The service can be built either locally in your IDE of choice or from the command line by running:
+`./gradlew clean build`
```

Comment:
Sorry one more thing to add: I would say here use `./gradlew clean assemble` command to build the application without running tests and then run `./gradlew clean build` to build the application as well as running the tests.  And just a side note that this will require standing up the PostgreSQL backend for the integration tests.